ledger-core/insolar/payload/marker_auth.go

```diff
@@ -38,7 +40,15 @@ func (m *VStateRequest) customSubject() (Reference, authSubjectMode) {
 }
 
 func (m *VCallResult) customSubject() (Reference, authSubjectMode) {
-	return m.GetCallee(), hasAuthSubject
+	switch m.CallType {
+	case CTMethod:
+		return m.GetCallee(), hasAuthSubject
+	case CTConstructor:
+		return m.GetCallOutgoing(), hasAuthSubject
```

Comment:
this looks weird to me ... why Callee is different for constructor?

Reply:
removed